app/views/admin/users/show.html.haml

```diff
@@ -0,0 +1,68 @@
+- content_for :page_title do
+  Viewing "#{ @user.username }"
+
+- content_for :body do
+  .ui.two.wide.column
+    .ui.row
+      = render partial: "menu", locals: { count: @count }
+
+  .ui.fourteen.wide.column
+    .ui.row
+      .ui.menu
+        = link_to edit_admin_user_path(@user), class: "item" do
+          %i.edit.icon
+          edit
+
+        .item
+          %i.unlock.alternative.icon
+          change password
+
+        -#.right.menu
+          -#.ui.simple.dropdown.item
+            -#more
+            -#%i.dropdown.icon
+            -#.menu
+
+              -#.divider
+              -#.header
+                -#%i.warning.sign.icon
+                -#DANGERZONE
+
+              -#.item{ data: { behavior: "modal", template: "admin/templates/users/delete", url: admin_user_path(@user), code: @user.code, title: @user.title } }
```

Comment:
> When Bart finds $1000, Lisa is able to convince him to take the family on a vacation to England instead of spending it on himself. Homer, however, ruins the holiday by crashing his rental British Mini Cooper into the Royal Carriage with the Queen inside. He is put on trial and eventually jailed. Meanwhile, Grandpa attempts to search for his English love interest.